engines/xeen/dialogs/dialogs_char_info.cpp

```diff
@@ -277,6 +278,17 @@ void CharacterInfo::addButtons() {
 	addPartyButtons(_vm);
 }
 
+int CharacterInfo::getRuDays(int val) {
+	int i = val % 100;
+	if (i < 5 || i > 20) switch (val % 10) {
```

Comment:
This is not following ScummVM formatting conventions. The if should have it's own {, and the switch should be on a new line.

Reply:
Fixed in later commits